src/main/java/codeu/controller/ServerStartupListener.java

```diff
@@ -41,4 +49,10 @@ public void contextInitialized(ServletContextEvent sce) {
 
   @Override
   public void contextDestroyed(ServletContextEvent sce) {}
+
+  private void initializeChatBot(){
+    String randomHash = BCrypt.hashpw(UUID.randomUUID().toString(), BCrypt.gensalt());
```

Comment:
I am using a random UUID to generate a password, but you can really use any random string of sufficient length